12-beginning-rxcocoa/projects/starter/Wundercast/Controllers/ApiController.swift

```diff
@@ -106,11 +106,15 @@ class ApiController {
 
   // MARK: - Api Calls
   func currentWeather(for city: String) -> Observable<Weather> {
-    buildRequest(pathComponent: "weather", params: [("q", city)])
-      .map { data in
-        try JSONDecoder().decode(Weather.self, from: data)
-      }
-  }
+  // Placeholder call
+  return Observable.just(
```

Comment:
Good catch!
We can shorten this to one of these two variations: 

```suggestion
  Observable.just(
```

```suggestion
  .just(
```

Besides, looks good. Thank you!